mpags-cipher.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+int main(int argc, char* argv[])
+{
+    const std::vector<std::string> cmdLineArgs{argv,argv+argc};
+
+    for(std::size_t i{0};i<cmdLineArgs.size();i++)
+    { 
+
+        if(cmdLineArgs[i] == "-h" or cmdLineArgs[i] == "--help")
+        {
+            std::cout << "Help Text" << std::endl;
+            continue;
```

Comment:
After giving help, you should exit the program as (generally) that's all the user will want the program to do. To do this here, use `return`.